build.ps1

```diff
@@ -131,6 +131,19 @@ Invoke-BuildStep 'Building NuGet.Clients projects - VS15 Toolset' {
     -skip:$SkipVS15 `
     -ev +BuildErrors
 
+## Building the VS15 NuGet.Tools.vsix for VS insertion
+Invoke-BuildStep 'Building NuGet.Tools.vsix for VS Insertion - VS15 Toolset' {
+        Build-ClientsProjectHelper `
+        -SolutionOrProject (Join-Path $NuGetClientRoot .\src\NuGet.Clients\NuGet.Tools\NuGet.Tools.csproj -Resolve)`
+        -Configuration $Configuration `
+        -ReleaseLabel $ReleaseLabel `
+        -BuildNumber $BuildNumber `
+        -Parameters @{'Insertable'='true'} `
+        -ToolsetVersion 15 `
+    } `
+    -skip:$SkipVS15 `
```

Comment:
I presume this build step should be running on CI only. If so then
`-skip:($SkipVS15 -or -not $CI)`